src/axelrod_dojo/utils.py

```diff
@@ -11,17 +10,20 @@
 ## Output Evolutionary Algorithm results
 
 class Outputer(object):
-    def __init__(self, filename, mode='w'):
-        self.output = open(filename, mode)
-        self.writer = csv.writer(self.output)
+    def __init__(self, filename, mode='a'):
+        self.file = filename
+        self.mode = mode
 
-    def write(self, row):
-        self.writer.writerow(row)
-        self.output.flush()
-        os.fsync(self.output.fileno())
+    def write_row(self, row):
+        with open(self.file, self.mode, newline='') as file_writer:
+            writer = csv.writer(file_writer)
+            writer.writerow(row)
+
+    def clear_file(self):
+        with open(self.file, 'w+', newline='') as file_writer:
```

Comment:
and here?

Reply:
As above, this would add a new line on the "empty" file.